<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>이미지 비교</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>◀</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>▶</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QListWidget" name="listWidget">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QListWidget" name="listWidget_2">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuvkdlf">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>정보</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menuvkdlf"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>폴더 열기</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>프로그램 정보 </string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>개발자 정보</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
